ucte/ucte-converter/src/main/java/com/powsybl/ucte/converter/UcteImporter.java

```diff
@@ -45,6 +45,12 @@
 
     private static final String[] EXTENSIONS = {"uct", "UCT"};
 
+    /**
+     * Key to store ucte power plant type in properties
+     * Concat this key with generator id
+     */
+    public static final String UCTE_POWERPLANT_TYPE_KEY = "UctePowerPlantType_";
```

Comment:
This property should be declared with the others: as it's used in both importer and exporter, the right place is UcteConstants.